ethsigner/core/src/test/java/tech/pegasys/ethsigner/core/jsonrpc/EeaSendTransactionJsonParametersTest.java

```diff
@@ -35,7 +35,7 @@
   @BeforeEach
   public void setup() {
     // NOTE: the factory has been configured as per its use in the application.
-    factory = TransactionFactory.createFrom(null, EthSigner.createJsonDecoder());
+    factory = new TransactionFactory(EthSigner.createJsonDecoder(), null);
```

Comment:
why null? Why not provide a default VertxNonceRequestTransmitterFactory here? Is transfactionfactory instance can work with null value?

Reply:
its not needed for these tests - in the same way we didn't used to provide a Web3j implementation